docs/python/configuration.md

```diff
@@ -0,0 +1,240 @@
+# Python Test Acceleration Configuration
+
+Python Test Acceleration works out of the box, but can be configured to take
+advantage of your specific environment.
+
+Have configuration needs not addressed here? Give us a holler at
+[hi@yourbase.io][hi].
+
+- Common Settings
+  - [`YOURBASE_ACCEPT_TOS`](#YOURBASE_ACCEPT_TOS)
+  - [`YOURBASE_ACTIVE_COHORT`](#YOURBASE_ACTIVE_COHORT)
+  - [`YOURBASE_COHORT_COUNT`](#YOURBASE_COHORT_COUNT)
+  - [`YOURBASE_LICENSE_KEY`](#YOURBASE_LICENSE_KEY)
+  - [`YOURBASE_OBSERVATION_MODE`](#YOURBASE_OBSERVATION_MODE)
+  - [`YOURBASE_REMOTE_CACHE`](#YOURBASE_REMOTE_CACHE)
```

Comment:
does python not have a `YOURBASE_LOCAL_CACHE` setting?

Reply:
Nope! It has a couple of different environment variables that give it similar behavior, but I want to replace them with `YOURBASE_LOCAL_CACHE` so I'm choosing not to document them.